<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8KC9AIJ" failures="4" tests="6" name="Test" time="51.245" errors="0" timestamp="2025-06-06T10:39:59 IST">
  <testcase classname="Testcases.TC_AccountRegistrationtest" name="verify_account_registration" time="2.333"/>
  <testcase classname="Testcases.TC_002LoginTest" name="testlogin" time="11.472">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at Testcases.TC_002LoginTest.testlogin(TC_002LoginTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testlogin -->
  <testcase classname="Testcases.TC003_LoginDDT" name="test_login_DDT" time="1.806"/>
  <testcase classname="Testcases.TC003_LoginDDT" name="test_login_DDT" time="11.146">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at Testcases.TC003_LoginDDT.test_login_DDT(TC003_LoginDDT.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login_DDT -->
  <testcase classname="Testcases.TC003_LoginDDT" name="test_login_DDT" time="10.8">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at Testcases.TC003_LoginDDT.test_login_DDT(TC003_LoginDDT.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login_DDT -->
  <testcase classname="Testcases.TC003_LoginDDT" name="test_login_DDT" time="0.514">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Testcases.TC003_LoginDDT.test_login_DDT(TC003_LoginDDT.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_login_DDT -->
</testsuite> <!-- Test -->
